java/test/resources/color.pde

```diff
@@ -1,2 +1,3 @@
+boolean test = true;
 color c1 = color(255, 255, 255);
-color c2 = #A011CD;
\ No newline at end of file
+color c2 = test ? #A011CD : #C0C0C0C0;
```

Comment:
Processing 3 currently doesn't allow the `#AARRGGBB` syntax. Was it mentioned somewhere that it should be added? (either way, you should be careful about it - `#00000000` is 100% opaque black because.. Processing)

Reply:
Thanks for flagging that! It's strange because some of my older processing sketches do have the full hex code with transparency but I do see that mainline processing does not support it (anymore?). I'll look into it. I also typically have used `processing-java` I wonder if we were in the situation of it being supported on one but not the other?